dashdotdb/models/dashdotdb.py

```diff
@@ -131,3 +143,27 @@ class ObjectKind(db.Model):
     name = db.Column(db.String(64), unique=False)
     deploymentvalidation = db.relationship('DeploymentValidation',
                                            backref='objectkind')
+
+
+class SLIType(db.Model):
+
+    __tablename__ = 'slitype'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(64), unique=False)
+    serviceslo = db.relationship('ServiceSLO', backref='slitype')
+
+
+class ServiceSLO(db.Model):
+
+    __tablename__ = 'serviceslo'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(64), unique=False)
+    value = db.Column(db.Integer, unique=False)
+    target = db.Column(db.Integer, unique=False)
+    slitype_id = db.Column(db.Integer, db.ForeignKey('slitype.id'))
+    token_id = db.Column(db.Integer, db.ForeignKey('token.id'))
+    service_id = db.Column(db.Integer, db.ForeignKey('service.id'))
+    cluster_id = db.Column(db.Integer, db.ForeignKey('cluster.id'))
```

Comment:
Sorry, missed this in the first round. No need for the cluster_id here. You get to the cluster via the namespace.

Reply:
Gotcha!